WooCommerce/Classes/Authentication/Store Creation/Progress/StoreCreationProgressViewModel.swift

```diff
@@ -0,0 +1,165 @@
+import UIKit
+
+final class StoreCreationProgressViewModel: ObservableObject {
+    enum Progress: Float, CaseIterable {
+        case creatingStore = 0.0
+        case buildingFoundations = 1.0
+        case organizingStockRoom = 2.0
+        case applyingFinishingTouches = 3.0
+        case finished = 4.0
+    }
+
+    @Published private(set) var progress: Progress = .creatingStore {
```

Comment:
nit: can this be private since it's not used by the view?
```suggestion
    @Published private var progress: Progress = .creatingStore {
```

Reply:
Done in cc6e6a370f61f3191741f40a8009abc1c5d52758